### DISCUSSION TITLE
Google does not index my repositories

### DISCUSSION THREAD

**Discussion by TimLiz** [community/community#34741] [New to GitHub]
Hey! I noticed just now, that google and other search engines do not see my profile & my repositories? Why? How can I fix it?

**Comment by jfullstackdev** [comment]
I answered same questions here, you may take a look 

https://github.com/community/community/discussions/30723#discussioncomment-3473908

https://github.com/community/community/discussions/31827#discussioncomment-3541620

https://github.com/community/community/discussions/22021#discussioncomment-3235171
